Packs/rasterize/Integrations/rasterize/rasterize.py

```diff
@@ -406,14 +522,24 @@ def ensure_chrome_running():  # pragma: no cover
             # There's no Chrome listening on that port, Start a new Chrome there
             demisto.debug(f"No Chrome found on port {chrome_port}")
             demisto.debug(f'Initializing a new Chrome session on port {chrome_port}')
-            browser, chrome_port = start_chrome_headless(str(chrome_port))
-            if browser:
-                return browser, chrome_port
+            return chrome_port
 
-        # There's already a Chrome listening on that port, Don't use it
 
-    demisto.error(f'Max retries ({MAX_CHROMES_COUNT}) reached, could not connect to Chrome')
-    return None, None
+def delete_row_with_old_chrome_configurations_from_info_file(info, instance_id, chrome_port):
+    index_to_delete = -1
+
+    info_without_new_lines = info.strip()
+    lines = info_without_new_lines.splitlines()
+    for index, line in enumerate(lines):
+        port_from_info, instance_id_from_info, chrome_options_from_info = line.strip().split('\t')
+        if port_from_info == chrome_port and instance_id_from_info == instance_id:
+            index_to_delete = index
+            break
+
+    if index_to_delete >= 0:
+        del lines[index_to_delete]
```

Comment:
What happens if you delete the last line and `lines` is an empty list?

Reply:
I think this case can't happen. Deleting a line in the file is only done when I find a line in the file of an instance with a different configuration from the configuration of the same instance I am now running with. That is, the deletion is performed only when there is at least one row.

Reply:
I see it's written in a different way now, so we're good.